### PR TITLE
Manifest update: GDATA.TypeRefHasher version 1.0.2

### PR DESCRIPTION
**Pull request by kdpuvvadi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34082)

### CODE REVIEW

File: manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+UpgradeBehavior: install
+Commands:
+- trh
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/GDATASoftwareAG/TypeRefHasher/releases/download/v1.0.2/trh.msi
+  InstallerSha256: 90F7ED58AF96932DE919FB59C7E7B387635EA1F8E3B3C88B29BF781F1EA862AD
```
Comment (ItzLevvie) [inline manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml:15]:
```suggestion
  InstallerSha256: 90F7ED58AF96932DE919FB59C7E7B387635EA1F8E3B3C88B29BF781F1EA862AD
  ProductCode: "{A8B2ABB6-B99F-4D50-BFD8-F1B02DDE26A3}"
  Scope: machine
```

File: manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GDATA.TypeRefHasher
+PackageVersion: 1.0.2
```
Comment (ItzLevvie) [inline manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.yaml [hunk 3]
```diff
@@ -1,24 +1,9 @@
...
-  InstallerSha256: 90F7ED58AF96932DE919FB59C7E7B387635EA1F8E3B3C88B29BF781F1EA862AD
-  InstallerType: msi
-PackageLocale: en-US
-ManifestType: singleton
-ManifestVersion: 1.0.0
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: GDATA.TypeRefHasher
+PackageVersion: 1.0.2
```
Comment (ItzLevvie) [inline manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: GDATA.TypeRefHasher
+PackageVersion: 1.0.2
```
Comment (ItzLevvie) [inline manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml (hunk 4 above)
Comment (kdpuvvadi) [reply manifests/g/GDATA/TypeRefHasher/1.0.2/GDATA.TypeRefHasher.installer.yaml:5]: it is version 1.0.2, isn't it?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> it is version 1.0.2, isn't it?

Control Panel shows `1.0.0` instead of `1.0.2`.

WinGet requires the `PackageVersion` to match Control Panel for the best application upgrade experience, otherwise, many users will end up in an upgrade loop when running the `winget upgrade GDATA.TypeRefHasher` command.

Please report the version number to the developer to be corrected in a future release.

**Pull request closed without merging**